### PR TITLE
Enumerable

### PR DESCRIPTION
**Pull request by AjroudiRida**
In this featured branch we did the following:
- Create a class MyList
- Create a module MyEnumerable
- implements the following methods : #all?, #any?, #filter

### CODE REVIEW

**Review by laiifuu — COMMENTED** [review]
# STATUS: INVALID :stop_sign: 

## Hi team :wave:  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

c60b0c9f441f523e67e64afc52bec690

## Invalid Code Review Request

You have submitted a project where the `README` file is empty.
Kindly make sure that you use the [`README` template](https://github.com/microverseinc/readme-template) before submitting for another code review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and :computer: Happy coding :computer: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by rloterh — COMMENTED** [review]
Hi Team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly ensure that your ReadMe is professionally documented: Let's ensure that provided placeholder texts/ guide instructions are removed and/or replaced with appropriate contents applicable under each section of your ReadMe.

**Kindly remove/update all aspects similar to the underlined:**
image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1,123 @@
...
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@AjroudiRida](https://github.com/ajroudirida)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (algerina) [inline README.md]: - [ ] This is a pair-programming activity, kindly add all team members here.

File: my_list.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  include MyEnumerable
+
+  def initialize(list)
+    @list = list
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
+end
```
Comment (algerina) [inline my_list.rb]: - [ ] You did a great work however when I try the app I have the following error:


Screenshot 2023-10-06 at 2 58 21 PM

**Review by algerina — CHANGES_REQUESTED** [review]
Hi @team ,

Great job so far! However you have some minor issues to work on.

May I highlight the following: 
-  Well  done. implementing all the classes✔
- Your code is well organized✔️



## CHANGES REQUIRED:muscle::clap:


Check the comments under the review.



 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Review by Munsa1 — APPROVED** [review]
Hi @AjroudiRida @ishimwepati , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Made Requested Changes :heavy_check_mark: 
image



## Optional suggestions
### ⚠️ Important ⚠️ 

- [ ] Please guys note that professional documentation is very key to software development no matter how trivial it may seem. Your readme file has no key features section and future features section and has broken links, such as having a deployment sub-section under the Getting Started section but having no corresponding link under the table of contents

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**